### PR TITLE
[Reviewer: Ellie] Install up to date virtualenv

### PR DESCRIPTION
**Pull request by johadalin**
The apt python-virtualenv package installs virtualenv 1.11.4. This is not up to date enough for our builds.

This instead uses pip to install the latest virtualenv package. At time of testing, this pulled down 15.1.0. The version in use on a number of machines is 13.1.0. 

@eleanor-merry Do you think we should pin to 13.1.0? The builds work with the latest version, but if the install command doesn't specify a version, it will not upgrade to a new version if someone has already installed the apt package.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -30,43 +30,49 @@ Pre-requisites
...
 
     ```
     sudo apt-get install python-pip python-dev python-virtualenv build-essential libffi-dev
     ```
 
-2. Lib-curl
+2. virtualenv
+
+   ```
+   sudo pip install virtualenv
```
Comment (eleanor-merry) [inline docs/development.md]: Should there be a version on this?

File: docs/development.md [hunk 2]
```diff
@@ -30,43 +30,49 @@ Pre-requisites
 
 Crest relies on APT packages available from the Project Clearwater repository server. To configure your build environment to install these packages, follow the instructions at [ReadTheDocs](http://clearwater.readthedocs.org/en/latest/Manual_Install.html#configure-the-apt-software-sources).
 
-1. Pip and virtualenv
+1. Pip and build tools
 
     ```
     sudo apt-get install python-pip python-dev python-virtualenv build-essential libffi-dev
```
Comment (eleanor-merry) [inline docs/development.md]: Can you remove python-virtualenv?

Comment (johadalin) [inline docs/development.md]: Thought I had. Clearly hadn't

**Comment by johadalin** [comment]
Updated to pin to version 13.1.0 as a known working point. This also ensures we upgrade any installation that is out of date, i.e. if someone has previously installed the apt-package.

**Pull request merged**